tools/jsl.node.conf

```diff
@@ -122,6 +122,7 @@
 +define setInterval
 +define setTimeout
 +define setImmediate
++define hasOwnProperty
```

Comment:
This is unneeded if you use o.hasOwnProperty() as suggested at L53 of test/unit/affinity.test.js